internal/services/executor/executor.go

```diff
@@ -1254,6 +1254,33 @@ func (e *Executor) tasksDataCleaner(ctx context.Context) error {
 	return nil
 }
 
+func (e *Executor) updateTimeToolboxLoop(ctx context.Context) {
+	for {
+		log.Debugf("updateTimeToolbox")
+
+		if err := e.updateTimeToolbox(ctx); err != nil {
+			log.Errorf("err: %+v", err)
+		}
+
+		sleepCh := time.NewTimer(24 * time.Hour).C
+		select {
+		case <-ctx.Done():
+			return
+		case <-sleepCh:
+		}
+	}
+}
+
+func (e *Executor) updateTimeToolbox(ctx context.Context) error {
+	now := time.Now()
+
+	if err := os.Chtimes("/tmp/agola/bin/agola-toolbox", now, now); err != nil {
```

Comment:
Use docker driver `d.initVolumeHostDir` + `agola-toolbox` instead of hardcoding the path.